packages/rest/src/validation/ajv-factory.provider.ts

```diff
@@ -55,34 +55,38 @@ export class AjvFactoryProvider implements Provider<AjvFactory> {
       // See https://github.com/epoberezkin/ajv#options
       validationOptions = {
         allErrors: true,
-        jsonPointers: true,
```

Comment:
How is this change going to affect existing LB4 applications? Does AJV v7 enable `jsonPointers` by default? If not, then how can LB4 users enable this feature themselves if they need it?